otter/convergence/service.py

```diff
@@ -172,7 +172,8 @@ def update_cache(group, servers, lb_nodes, now):
         sd = thaw(server.json)
         if is_autoscale_active(server, lb_nodes):
             sd["_is_as_active"] = True
-        server_dicts.append(sd)
+        if server.state != ServerState.DELETED or include_deleted:
```

Comment:
Duh. I didn't realize it will not add non-deleted servers. This is fine. If I was to write I would probably do
```python
if not include_deleted and server.state == ServerState.DELTED:
   continue
```
in beginning of the loop which probably increases the code size. But all this is bike-shedding :). This is fine as it is.